web/src/components/nomination/NominateForm.vue

```diff
@@ -48,40 +59,91 @@ export default {
     NameInput,
     JobInput,
     LocationInput,
+    UrlsInput,
     SubmitterInput,
     TopicsInput,
   },
   data() {
     return {
       form: {
-        speakerName: {
-          title: '',
-          first: '',
-          middle: '',
-          last: '',
+        name: {
+          en: '',
+          ja: '',
         },
-        speakerEmail: '',
-        firstJob: {
+        email: '',
+        photo_url: '',
+        job: {
           title: '',
           company: '',
         },
-        speakerBio: '',
+        speaker_bio: '',
```

Comment:
Are we moving to snake case because airtable expects it?

Reply:
Yeah, just on the data layer that interacts with the database 🙏